Presentation/PresentationStyle.swift

```diff
@@ -99,7 +99,35 @@ public extension PresentationStyle {
             return from.modallyPresentQueued(vc, options: options) {
                 Future { completion in
                     let bag = DisposeBag()
-                    bag += viewController.installDismissButton().onValue { completion(.failure(PresentError.dismissed)) }
+
+                    // The presentationController of an alert controller should not have its delegate modified
+                    if !(vc is UIAlertController) {
+                        let customPresentationDelegate: CustomAdaptivePresentationDelegate
+                        if let givenDelegate = viewController.customAdaptivePresentationDelegate {
+                            customPresentationDelegate = givenDelegate
```

Comment:
Should we rely on someone else holding a reference for us? If not, potential simplification:

let delegate = viewController.customAdaptivePresentationDelegate ?? CustomAdaptivePresentationDelegate()
bag.hold(delegate)